cdk/java/README.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to your CDK Java project!
+
+This is a blank project for CDK development with Java.
```

Comment:
Is it? 😆

Reply:
Please check the `Language specific prerequisites` section we have in the other patterns (Go, Python, TypeScript, etc.)